### ISSUE TITLE
SqrSNRGeometricFactorHist: "md5sum is obsolete"

### ISSUE THREAD

**Issue by dbkeitel** [octapps/octapps#45]
@ReinhardPrix first time I ran this in a long time, with octave 4.4.1: 
```
warning: md5sum is obsolete and will be removed from a future version of Octave, please use hash instead
warning: called from
    md5sum at line 50 column 5
    SqrSNRGeometricFactorHist at line 103 column 7
    SensitivityDepthStackSlide at line 134 column 8
    estimate_sensitivity_octapps at line 46 column 7
    __octapps_run_driver__ at line 226 column 5
```

**Comment by ReinhardPrix** [comment]
ok, thanks for the heads-up, David!
I mostly stick to octave-4.0.3 to make sure everything keeps working especially swig etc.
Now just replacing md5sum() with hash() wouldn't work for me, unfortunately, as that function seems to be newer and doesn't exist in octave-4.0.3 yet. So one would need to make a version-checking replacement that picks either md5sum or hash dependent on the octave version ...

**Comment by dbkeitel** [comment]
Since our user base is fairly small, I suggest we stick with what we have for now until it is actually removed from the default version on one of our preferred OSs...

I also saw a few other deprecation warnings for `isstr` from across the package (`please use ischar instead`).

**Comment by kwwette** [comment]
Fixed in #46
